frontend/css/rodi_css.css

```diff
@@ -2404,7 +2404,10 @@ h2.odfri_title_red {
     z-index: 100;
 }
 
-
 .text-content {
     font-size: 17px;
+}
+
+tr.ng-scope {
```

Comment:
You mean `tr.ng-click`?

Reply:
Oh, I discovered that `.ng-scope` is a class applied by Angular: https://docs.angularjs.org/guide/scope#scope-hierarchies

I wonder why I could not find it in the source code.

I still think `[ng-click]` would reflect better our intent.